doc/MAC_SETUP.md

```diff
@@ -20,6 +20,13 @@ If you have an older version of postgres, `brew postgresql-upgrade-database`
 
 For a more GUI focused postgres experience, try [Postgres.app](https://postgresapp.com/) an alternative to the CLI focused default postgres
 
+If you are having trouble connecting to your local postgres database using pgAdmin or another local tool, try the following configuration:
```

Comment:
Where is this configuration? pgAdmin or `database.yml`? Please add which configuration to alter. If `database.yml`, please add a note not to commit changes. I also don't know what "Maintenance Database" means in that context.